Mathlib/Topology/PartialHomeomorph.lean

```diff
@@ -1051,37 +1051,37 @@ def prod (e : PartialHomeomorph α β) (e' : PartialHomeomorph γ δ) :
 #align local_homeomorph.prod PartialHomeomorph.prod
 
 @[simp, mfld_simps]
-theorem prod_symm (e : PartialHomeomorph α β) (e' : PartialHomeomorph γ δ) :
+theorem prod_symm (e : PartialHomeomorph X Y) (e' : PartialHomeomorph Z Z') :
     (e.prod e').symm = e.symm.prod e'.symm :=
   rfl
 #align local_homeomorph.prod_symm PartialHomeomorph.prod_symm
 
 @[simp]
-theorem refl_prod_refl {α β : Type*} [TopologicalSpace α] [TopologicalSpace β] :
-    (PartialHomeomorph.refl α).prod (PartialHomeomorph.refl β) = PartialHomeomorph.refl (α × β) :=
+theorem refl_prod_refl :
+    (PartialHomeomorph.refl X).prod (PartialHomeomorph.refl Y) = PartialHomeomorph.refl (X × Y) :=
   PartialHomeomorph.ext _ _ (fun _ => rfl) (fun _ => rfl) univ_prod_univ
 #align local_homeomorph.refl_prod_refl PartialHomeomorph.refl_prod_refl
 
 @[simp, mfld_simps]
-theorem prod_trans {η : Type*} {ε : Type*} [TopologicalSpace η] [TopologicalSpace ε]
-    (e : PartialHomeomorph α β) (f : PartialHomeomorph β γ) (e' : PartialHomeomorph δ η)
-    (f' : PartialHomeomorph η ε) : (e.prod e').trans (f.prod f') = (e.trans f).prod (e'.trans f') :=
+theorem prod_trans (e : PartialHomeomorph X Y) (f : PartialHomeomorph Y Z)
+    (e' : PartialHomeomorph Z' Z'') (f' : PartialHomeomorph Z'' Z''') :
```

Comment:
... and with `PartialHomeomorph Z Z'` here.

Reply:
I have used `PartialHomeomorph X Y`, `PartialHomeomorph Y Z` and then `PartialHomeomorph X' Y'` and `PartialHomeomorph Y' Z'`.